helpers/labgrid-raw-interface

```diff
@@ -67,6 +67,15 @@ def main(program, ifname, count):
             args.append("-c")
             args.append(str(count))
 
+        if duration:
+            # The duration is implemented by specifying the number of seconds
+            # before rotating the dump file, but limiting the number of files
+            # to 1
+            args.append("-G")
+            args.append(str(duration))
+            args.append("-W")
+            args.append("1")
```

Comment:
I suspect this might fail in the future when used in combination with writing to stdout. Handling that in the helper would mean using a subprocess instead `os.execvp`, so that can wait util that actually is a problem.

Reply:
Agreed, let's keep it like this until it becomes a problem.